### PR TITLE
Fix Bug 1163628, switch research links to point to DDG

### PR DESCRIPTION
**Pull request by schalkneethling**
@alexgibson r?

### CODE REVIEW

**Comment by pmclanahan** [comment]
r+ from me.

File: media/js/plugincheck/check-plugins.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ $(function() {
     };
     var mediaURL = window.trans('media-url') + 'img/plugincheck/app-icons/';
     var unknownPluginUrl = function (pluginName) {
-        return 'https://www.google.com/search?q=' + encodeURI(window.trans('googleSearchq') + ' ' + pluginName);
+        return 'https://duckduckgo.com/?q=' + encodeURI(window.trans('googleSearchq') + ' ' + pluginName);
```
Comment (alexgibson) [inline media/js/plugincheck/check-plugins.js]: If we're gonna do this let's remove all references to the goog `googleSearchq`

Comment (schalkneethling) [inline media/js/plugincheck/check-plugins.js]: Ah yes, good point indeed.

**Comment by schalkneethling** [comment]
@alexgibson updated, thanks!

**Comment by alexgibson** [comment]
r+ :cake:

**Pull request merged**